applications/asset_tracker_v2/tests/json_common/src/main.c

```diff
@@ -1166,170 +1184,30 @@ static void test_floating_point_encoding_configuration(void)
 	cJSON *accel_inact_timeout =
 		cJSON_GetObjectItem(decoded_config_obj, CONFIG_ACC_INACT_TIMEOUT);
 
-	zassert_not_null(accel_act_thresh, "Accelerometer activity threshold is NULL");
-	zassert_not_null(accel_inact_thresh, "Accelerometer inactivity threshold is NULL");
-	zassert_not_null(accel_inact_timeout, "Accelerometer inactivity timeout is NULL");
+	TEST_ASSERT_NOT_NULL(accel_act_thresh);
+	TEST_ASSERT_NOT_NULL(accel_inact_thresh);
+	TEST_ASSERT_NOT_NULL(accel_inact_timeout);
 
 	decoded_values.accelerometer_activity_threshold = accel_act_thresh->valuedouble;
 	decoded_values.accelerometer_inactivity_threshold = accel_inact_thresh->valuedouble;
 	decoded_values.accelerometer_inactivity_timeout = accel_inact_timeout->valuedouble;
 
-	zassert_within(decoded_values.accelerometer_activity_threshold,
-		       data.accelerometer_activity_threshold, 0.1,
-		       "Decoded value is not within delta");
+	TEST_ASSERT_DOUBLE_WITHIN(0.1, data.accelerometer_activity_threshold,
+				decoded_values.accelerometer_activity_threshold);
 
-	zassert_within(decoded_values.accelerometer_inactivity_threshold,
-		       data.accelerometer_inactivity_threshold, 0.1,
-		       "Decoded value is not within delta");
+	TEST_ASSERT_DOUBLE_WITHIN(0.1, data.accelerometer_inactivity_threshold,
+				decoded_values.accelerometer_inactivity_threshold);
 
-	zassert_within(decoded_values.accelerometer_inactivity_timeout,
-		       data.accelerometer_inactivity_timeout, 0.1,
-		       "Decoded value is not within delta");
+	TEST_ASSERT_DOUBLE_WITHIN(0.1, data.accelerometer_inactivity_timeout,
+				decoded_values.accelerometer_inactivity_timeout);
 
 	cJSON_Delete(decoded_root_obj);
 }
 
-/* Setup and teardown functions. Used to allocate root and array objects used in test and to
- * cleanup allocated memory afterwards.
- */
-
-static void test_setup_object(void)
-{
-	dummy.root_obj = cJSON_CreateObject();
-	zassert_not_null(dummy.root_obj, "Root object is NULL");
-}
 
-static void test_teardown_object(void)
-{
-	cJSON_FreeString(dummy.buffer);
-	cJSON_Delete(dummy.root_obj);
-}
-
-static void test_setup_array(void)
-{
-	dummy.array_obj = cJSON_CreateArray();
-	zassert_not_null(dummy.root_obj, "Root object is NULL");
-}
-
-static void test_teardown_array(void)
-{
-	cJSON_FreeString(dummy.buffer);
-	cJSON_Delete(dummy.array_obj);
-}
-
-void test_main(void)
+int main(void)
 {
 	cJSON_Init();
-
-	ztest_test_suite(json_common,
```

Comment:
This whole block is no longer needed as unity framework automatically picks up the test cases by searching this file for `test_` functions.